llvm/lib/Target/Sparc/SparcInstrVIS.td

```diff
@@ -309,4 +309,11 @@ def : Pat<(i64 (ctlz i64:$src)), (LZCNT $src)>;
 // The zero extension will leave us with 32 extra leading zeros,
 // so we need to compensate for it.
 def : Pat<(i32 (ctlz i32:$src)), (ADDri (LZCNT (SRLri $src, 0)), (i32 -32))>;
+
+def : Pat<(i32 (bitconvert f32:$src)), (MOVSTOUW $src)>;
+def : Pat<(i64 (zext (i32 (bitconvert f32:$src)))), (MOVSTOUW $src)>;
+def : Pat<(i64 (sext (i32 (bitconvert f32:$src)))), (MOVSTOSW $src)>;
```

Comment:
Should handle `anyext` as well. (There is a combined `zanyext` pattern that might be useful.)